opsramp/service_maps.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+#
+# A minimal Python language binding for the OpsRamp REST API.
+#
+# sites.py
+# Classes dealing directly with OpsRamp Sites.
+# Sites are used to organise resources by geographical location.
+#
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+from opsramp.base import ApiWrapper
+
+
+class Service_Maps(ApiWrapper):
```

Comment:
conventionally would not have _ in a python class name. more typical like ServiceMaps

Reply:
fixed